app/views/obituaries/_form.html.erb

```diff
@@ -36,23 +36,25 @@
             </div>
           </div>
 
-          <div class="form-group">
+          <div class="form-label-group row">
+            <div class="col">
             <%= f.label :description, class: 'col-form-label-lg'%>
-            <%= f.text_field :description, :size => 150, class: "form-control" %>
+            <%= f.text_area :description, cols: 5, rows: 10, class: "form-control" %>
+            </div>
           </div>
 
           <div class="form-group" id="age-select">
             <%= f.label :age, class: 'col-form-label-lg'%>
             <%= f.select :age, options_for_select((0..150), 100), { }, {class: 'custom-select' } %>
           </div>
 
-          <div class="form-check" id="covid-select">
-            <%= f.label :covid, "COVID-19", class: 'col-form-label-lg' %>
-            <%= f.check_box :covid  %>
+          <div class="form-check row" id="covid-select">
```

Comment:
turns out the word "row" is really important for getting everything to line up!

Reply:
good to know

Reply:
Found this out last night too